import java.lang.Exception
import java.lang.IllegalStateException
import kotlin.random.Random

fun main(args: Array<String>) {
//    val username = readLine()?:""
//    val currentYear = readLine()?:""
//    println(currentYear.toInt() - username.toInt())
//    println(Random.nextInt(20,30))
//    val price =29.99
//    val var1 = null
//    val purchase = var1?.toInt()?:3
//    val var2 = var1.toDouble()
//    println("In 5 yrs, ${var2 * 5.5 * 5/ 100}")
//   var catName: String? = null
//    val var2 = var1?.toDouble()
////    val var3 = var2?.times(7).toString()
//    val test = readLine()
////    throw IllegalStateException("none")
//    try {
//        println(test?.toInt()?.times(9.99))
//    } catch (e: Exception){
//        e.printStackTrace()
//        println("An exception happened \n ${e.localizedMessage}")
//    } finally {
//        println("The execution has happened")
//    }

    val colors = listOf("blue","red","yellow")
    val colors2 = listOf<String>()
    println(colors.size)
}








